docs/oss/issues.md

```diff
@@ -23,9 +23,9 @@ Here all requests for Eclipse infrastructure  go in.
 
```

Comment:
Above this there are a few remarks:

L17: perhaps: "tractus-x: " vs. "tractus-x"
L10: ""Put there" sounds weird to me
L15: Perhaps example what this could be like github repos, jenkins etc.